src/components/motion/MotionController.h

```diff
@@ -1,8 +1,12 @@
 #pragma once
 
 #include <cstdint>
-#include <drivers/Bma421.h>
-#include <components/ble/MotionService.h>
+
+#include <FreeRTOS.h>
+#include <portmacro_cmsis.h>
```

Comment:
Why do we need this include (asking because I don't think we use it anywhere else in the code)? And should'nt we include `portmacro.h` instead?

Reply:
That is the file that defines `TickType_t`, and my editor just automatically added an include for it. It seems that `FreeRTOS.h` includes `portable.h` which includes `portmacro.h` which includes `portmacro_cmsis.h`. Because of this, it doesn't need to be explicitly included in our code, and I think it's better to remove it.